app/refactor/viewmodels/p5/package.scala

```diff
@@ -386,6 +386,23 @@ package object p5 {
     ).commaSeparate
   }
 
+  implicit class RichConsignmentItemType04(value: ConsignmentItemType04) {
+
+    // TODO check if we can use N/A for absent optional values
```

Comment:
Does this need to be actioned in this PR, or is it a question for later?

Reply:
This is just to generate the pdf, nothing to do with populating it.